aria_data_loaders/aria_data_utils/spot_aria_p1_node.py

```diff
@@ -0,0 +1,336 @@
+import logging
+from typing import Any, Dict, List, Optional, Tuple
+
+import click
+import numpy as np
+import rospy
+import sophus as sp
+from aria_data_utils.conversions import (
+    ros_PoseStamped_to_sp_SE3,
+    ros_TransformStamped_to_sophus_SE3,
+    sophus_SE3_to_ros_PoseStamped,
+    sophus_SE3_to_ros_TransformStamped,
+    xyt_to_sophus_SE3,
+)
+from geometry_msgs.msg import PoseStamped
+from spot_rl.envs.skill_manager import SpotSkillManager
+from spot_wrapper.spot import Spot
+from spot_wrapper.spot_qr_detector import SpotQRDetector
+from std_msgs.msg import Bool
+from tf2_ros import (
+    ConnectivityException,
+    ExtrapolationException,
+    LookupException,
+    StaticTransformBroadcaster,
+)
+from tf2_ros.buffer import Buffer
+from tf2_ros.transform_listener import TransformListener
+
+
+class SpotAriaP1Node:
+    """
+    TODO: Add docstring
+    """
+
+    def __init__(self, spot: Spot, verbose=False):
+        # Get Spot Skill Manager
+        self.skill_manager = SpotSkillManager(
+            should_stand_on_start=False,
```

Comment:
Remove this flag, add spot.nav & spot.sit logic here which brings robot next to dock in a way it sees the QR marker

Reply:
added